eslint.config.mjs

```diff
@@ -0,0 +1,10 @@
+import { storyblokLintConfig } from '@storyblok/eslint-config';
+
+export default storyblokLintConfig({
+  rules: {
+    '@typescript-eslint/no-this-alias': 'off',
```

Comment:
@edodusi Are these rules definite, or temporary? If they're temporary, I suggest we add to-do comment, on the lines of `// @TODO: remove after fixing in v7`

Reply:
they're temporary, will add the comment!